.travis.yml

```diff
@@ -1,21 +1,19 @@
 language: python
-
-# run tests for all envs
+dist: jammy
 python:
-  - 3.7
-  - 3.8
-  - 3.9
-  - 3.10
-  - 3.11
-  - 3.12
-env: ISOLATED=false
-install: pip install tox tox-travis
-script: tox
+  - '3.12'
+  - '3.11'
+  # - '3.10' # Tries to install rpds-py from source and fails.
+  - '3.9'
+  - '3.8'
+  - '3.7'
+install: pip install -U 'jsonschema>=4.0.0' coverage
+script: |
+  coverage run --source=genson -m unittest
+  coverage report --omit='*/__main__.py' --fail-under=90
```

Comment:
Duplicating this from tox.ini isn't ideal, but tox was causing failures in older versions, so I had to rip it out, and now there isn't really a good way around the duplication.

This does speed up the tests a little though because there's fewer steps and dependencies to running them.